contribute/design/README.md

```diff
@@ -1,61 +1,57 @@
 
-
-## OpenEBS Control Plane (aka OpenEBS Storage Orchestration, aka maya)
-
-OpenEBS Control Plane - auguments the functionality provided by the Container Orchestrator, with storage specific orchestration capabilities, via a set of services. OpenEBS control plane can hook into any of the container orchestrators like Kubernetes, Docker Swarm, Nomad, etc., making it possible to run OpenEBS hyper-converged with Container Orchestrators. 
+OpenEBS Control Plane - augments the functionality provided by the Container Orchestrator, with storage-specific orchestration capabilities, via a set of services. OpenEBS control plane can hook into any of the container orchestrators like Kubernetes, Docker Swarm, Nomad, etc., making it possible to run OpenEBS hyperconverged with Container Orchestrators. 
 
 ![OpenEBS Control Plane](../../documentation/source/_static/OpenEBS-Control-Plane.png)
 
 OpenEBS Control Plane is also delivered as micro-services, where the services built can be classified as follows:
-- Plugins into Container Orchestrotors, which augument the functionality by hooking into the framework provided like:
+- Plugins into Container Orchestrators, which augment the functionality by hooking into the framework provided like:
   - K8s Dynamic Provisioners - [openebs-provisioner](https://github.com/openebs/external-storage/tree/master/openebs)
   - Kube Dashboard - [openebs-dashboard](https://github.com/openebs/dashboard)
   - Extended Schema - like CRDs in case of Kubernetes to store the OpenEBS specific configuration related data
-- Cluster Services, provides OpenEBS specific storage intelligence like:
-  - maya-apiserver - includes the API for performing Volume operations that can translate requests into CO specific operations
-  - maya-mulebot - uses the information gathered to suggest optimized placement and event handling tips
-  - maya-connect - allows to upload the monitoring data to maya-cloud for further storage access pattern analysis
-- Node Services, provides OpenEBS specific storage intelligence running along-side kubelet like:
-  - maya-agent - includes storage management functionality 
+- Cluster Services provides OpenEBS specific storage intelligence like:
+  - Maya-apiserver - includes the API for performing Volume operations that can translate requests into CO specific operations
+  - Maya-mulebot - uses the information gathered to suggest optimized placement and event handling tips
+  - Maya-connect - allows uploading the monitoring data to maya-cloud for further storage access pattern analysis
+- Node Services provides OpenEBS specific storage intelligence running along-side kubelet like:
+  - Maya-agent - includes storage management functionality 
 
-Monitoring and Tracing capabilities are added by instrumenting the above services with prometheus, heapster, grafana and jaegar. 
+Monitoring and Tracing capabilities are added by instrumenting the above services with prometheus, heapster, grafana, and jaegar. 
```

Comment:
prometheus = Prometheus
heapster = Heapster
grafana = Grafana
jaegar = Jaegar

Reply:
We sure about this ? @kmova

Reply:
Let us do it, just remove the Heapster from the list.

Reply:
Removed Heapster